src/app/panels/terminal.js

```diff
@@ -461,10 +509,6 @@ class Terminal {
       self._output.apply(self, args)
     }
   }
-  log () {
```

Comment:
can we keep it for now? `TxLogger` is using that function

Reply:
oh i see.
i'm anyway restructuring again. I'll update this PR soon.
I'm optimizing the log structure, so that it makes the filtering easy.
While developing, it easily happens, that my terminal fills with 10k or 100k ...i even had already 1million transaction messages... and that make it slow, so i have to tune this a bit, especially when you want to go and filter them